install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,26 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: CannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates since {{ "{{ $labels.lastTime }}" }}. Failure reason {{ "{{ $labels.reason }}" }}. {{ "{{ $labels.message }}" }}.
```

Comment:
Do we want to... link [here][1]?  To docs for the specific reason?  Pin to `release-4.6`?  Float with master?  Not link to docs at all?  Suggest:

> Clear ClusterVersion.spec.channel if you have no upstream update recommendation service available.

I don't know if there's a cap on alert message text length or not, but I'd like to provide as much actionable context here as possible, so users are less likely to say "huh, another critical alert I know nothing about" and walk away without fixing it.

[1]: https://github.com/openshift/cluster-version-operator/blob/master/docs/user/status.md#retrievedupdates

Reply:
Done with 06c550ff9.